### PR TITLE
[17.0][MIG] product_pricelist_fixed_currency_rate: Migration to 17.0

### PR DESCRIPTION
**Pull request by kobros-tech**
I couldn't add my name in README file, but I added it in contributors file

### CODE REVIEW

**Comment by sanderlienaerts** [comment]
Hi @kobros-tech, please make sure to run pre-commit before migrating. The changes made by pre-commit should come in front of your migration commit.

While you are migrating, you can add yourself as contributor in the readme folder and the contributor file. Pre-commit will run again during the pre-commit phase of your migration commit and one of the actions it will execute is to prepare the readme and module info.  

I haven’t looked at your changes, but will do once the commit with the pre-commit changes is present.

**Comment by kobros-tech** [comment]
thanks for replying

I already tried am pre commit many times but it fails.

It genertes the readme file without my name as contributer.

the error is about .deb file and ar language as I think.

after pulling request 3 tests fail.

I am not aware about am and package management and I got exhausted really.

**Comment by kobros-tech** [comment]
I got these tests error, but I don't understand and don't know how to solve

https://github.com/OCA/product-attribute/actions/runs/10269617361/job/28415457597

https://github.com/OCA/product-attribute/actions/runs/10269617350/job/28415457560

File: product_pricelist_fixed_currency_rate/__init__.py [hunk 1]
```diff
@@ -1 +1,2 @@
 from . import models
+from .hooks import pre_init_hook, post_init_hook, uninstall_hook, post_load
```
Comment (sanderlienaerts) [inline product_pricelist_fixed_currency_rate/__init__.py]: You can see the changes you've made in commit https://github.com/OCA/product-attribute/pull/1702/commits/7c1ef7a91da0209ae6154c0df995d8f06c31a894. The line this comment is on, was added - yet there's no hooks folder and there are no hooks implemented for this module. To fix the issue you are having now with the tests, remove this line.

File: product_pricelist_fixed_currency_rate/views/pricelist_views.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <field name="inherit_id" ref="product.product_pricelist_item_form_view" />
+        <field name="arch" type="xml">
+            <field name="base_pricelist_id" position="after">
+                <field name="is_fixed_currency_rate_applicable" invisible="1" />
+                <field name="do_inverse_currency_rate" invisible="1" />
+                <field
+                    name="currency_rate_tooltip"
+                    nolabel="1"
+                    attrs="{
+                    'invisible': [('is_fixed_currency_rate_applicable', '=', False)]}"
```
Comment (sanderlienaerts) [inline product_pricelist_fixed_currency_rate/views/pricelist_views.xml:17]: The migration tasks have not been done. Please follow the tasks as mentioned in: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#tasks-to-do-in-the-migration

**Review by sanderlienaerts — CHANGES_REQUESTED** [review]

**Comment by sanderlienaerts** [comment]
Before migrating, to not make things more difficult, please make sure to follow the steps in _[Technical method to migrate a module from "16.0" to "17.0" branch --> Full process](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#tasks-to-do-in-the-migration)_. This ensures the pre-commit changes you need to do are present before the actual migration commit. Or maybe you should consider using the [`oca_port`](https://github.com/OCA/oca-port) migration tool.

**Comment by kobros-tech** [comment]
Hello Sander!

I am grateful to you for your advice.
I did what you have suggested and I still suffer from the same problem.

I took a snapshot to describe what is happening.

**Comment by kobros-tech** [comment]
also my name is not appened to the contributors q

**Pull request closed without merging**